doc/mcuboot/readme-ncs.rst

```diff
@@ -26,6 +26,6 @@ See the `MCUboot output build files`_ page for a list of all these files.
 
 .. note::
    When you use MCUboot in the direct-xip mode, enable the ``SB_CONFIG_MCUBOOT_BUILD_DIRECT_XIP_VARIANT`` sysbuild Kconfig option to let the build system generate an additional set of files for the second application slot.
-   These files are identical to the ones listed on the `MCUboot output build files`_ page, but they are placed in the :file:`mcuboot_secondary_app` folder.
-   For example, :file:`mcuboot_secondary_app/zephyr/zephyr.signed.bin` is created and placed in the second slot on the target device when the :file:`zephyr.signed.bin` file is placed in the first slot.
+   These files are identical to the ones listed on the `MCUboot output build files`_ page, but they are placed in the :file:`mcuboot_secondary_app` folder for the main application in the :file:`<application>_secondary_app` folder.
```

Comment:
This is a bit unclear, something's missing. Should it say: 
``they are placed in the :file:`mcuboot_secondary_app` folder, while the files for the main application are located in the :file:`<application>_secondary_app` folder.`` ?

Reply:
No idea how that happened.
Rephrased into:
```
These files are identical to the ones listed on the `MCUboot output build files`_ page, but they are placed in the :file:`mcuboot_secondary_app` folder for the main application, while the files for the remaining applications are located in the respective :file:`<application>_secondary_app` folder.
```